src/libstd/net/tcp.rs

```diff
@@ -630,6 +630,8 @@ impl TcpListener {
     /// listener.set_only_v6(true).expect("Cannot set to IPv6");
     /// ```
```

Comment:
According to @sfackler, the `set_only_v6` call will always fail. Maybe the doc comment above should be removed as it's not accurately describing its actual behavior?